pom.xml

```diff
@@ -125,24 +125,50 @@
         <maven.compiler.target>${java.version}</maven.compiler.target>
 
 		<log4j.version>1.2.16</log4j.version>
-		<spring.version>4.3.19.RELEASE</spring.version>
+		<spring.version>5.1.7.RELEASE</spring.version>
 		<spring.groupId>org.springframework</spring.groupId>
-		<slf4j.version>1.7.10</slf4j.version>
-		<gigaspaces.version>14.0.1</gigaspaces.version>
+		<slf4j.version>1.7.26</slf4j.version>
+		<gigaspaces.version>14.5.0</gigaspaces.version>
 		<rxjava.version>1.3.8</rxjava.version>
 		<hystrix.version>1.5.18</hystrix.version>
 		<archaius.version>0.4.1</archaius.version>
-		<jackson2.version>2.8.11</jackson2.version>
-		<guava.version>20.0</guava.version>
-		<junit.version>4.11</junit.version>
-		<hamcrest.version>1.2.1</hamcrest.version>
-		<mockito.version>1.9.0</mockito.version>
+		<jackson2.version>2.11.2</jackson2.version>
+		<guava.version>29.0-jre</guava.version>
+		<junit.version>4.12</junit.version>
+		<hamcrest.version>1.3</hamcrest.version>
+		<mockito.version>2.23.0</mockito.version>
 		<lifecycle-mapping.version>1.0.0</lifecycle-mapping.version>
 		<dropwizard.version>3.1.2</dropwizard.version>
-		<netty.version>4.0.31.Final</netty.version>
-		<gs-test.version>14.0.8</gs-test.version>
+		<netty.version>4.1.30.Final</netty.version>
 		<mimer-config.version>0.0.4</mimer-config.version>
 		<hystrix-multiconfig.version>0.0.3</hystrix-multiconfig.version>
+		<gs-test.groupId>com.avanza.gs14-5</gs-test.groupId>
+		<gs-test.version>14.5.0</gs-test.version>
+
+		<!-- Plugins -->
+		<maven-shade-plugin.version>2.4.1</maven-shade-plugin.version>
+		<maven-assembly-plugin.version>2.2.1</maven-assembly-plugin.version>
+		<maven-clean-plugin.version>2.5</maven-clean-plugin.version>
+		<maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+		<maven-dependency-plugin.version>2.8</maven-dependency-plugin.version>
+		<maven-deploy-plugin.version>2.8.1</maven-deploy-plugin.version>
+		<maven-install-plugin.version>2.3.1</maven-install-plugin.version>
+		<maven-jar-plugin.version>2.3.1</maven-jar-plugin.version>
+		<maven-javadoc-plugin.version>3.2.0</maven-javadoc-plugin.version>
+		<maven-rar-plugin.version>2.2</maven-rar-plugin.version>
```

Comment:
Maybe this is another plugin that we could remove?

Reply:
Good idea! @oskarwiksten I will remove it and check the other plugins and verisons while I'm at it